content/post/stash-v2024.12.18/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Introducing Stash v2024.12.18
+date: "2024-12-18"
+weight: 10
+authors:
+- Md Anisur Rahman
+tags:
+- backup
+- cli
+- disaster-recovery
+- kubedump
+- kubernetes
+- mongodb
+- postgresql
+- restore
+- stash
+---
+
+We are pleased to announce the release of [Stash v2024.8.27](https://stash.run/docs/v2024.8.27/setup/), packed with new features. You can check out the full changelog [HERE](https://github.com/stashed/CHANGELOG/blob/master/releases/v2024.4.8/README.md). In this post, we'll highlight the changes done in this release.
+
+### New Features
+
+#### Introducing the `multiDumpArgs` Parameter for MySQL Backups
+
+We are excited to introduce the `multiDumpArgs` parameter, which is available for all MySQL versions during the backup process.
+
+**What is `multiDumpArgs`?**
+
+The `multiDumpArgs` parameter can be specified under the `spec.task.params` section in the `BackupConfiguration`. It allows you to define multiple dump commands, which will be executed sequentially and separated by the Bash `&&` operator.
```

Comment:
It allows you to provide arguments for multiple dump commands, which will be executed sequentially.